pipeline_dp/dp_engine.py

```diff
@@ -180,6 +187,19 @@ def _drop_not_public_partitions(self, col, public_partitions,
             f"Public partition selection: dropped non public partitions")
         return self._ops.map_tuple(col, lambda k, v: v, "Drop key")
 
+    def _add_empty_public_partitions(self, col, public_partitions,
+                                     aggregator_fn):
+        self._add_report_stage(
+            "Adding empty accumulators to public partitions and joining "
```

Comment:
Those reports are for API users. Accumulators are not part of the API. 
Maybe something like that
"Adding empty partitions for those public partitions which are missing in data" 

Technically speaking right here we're doing in slightly different way, but it's equavalent. WDYT?